unity-client/Assets/Scripts/MainScripts/DCL/Controllers/CatalogController/CatalogController.cs

```diff
@@ -306,22 +309,23 @@ private void CheckForRequestsByContextTimeOuts()
     {
         if (pendingWearablesByContextRequestedTimes.Count > 0)
         {
-            List<string> expiredRequestedTimes = new List<string>();
+            List<string> expiredRequests = new List<string>();
             foreach (var promiseByContextRequestedTime in pendingWearablesByContextRequestedTimes)
             {
-                if ((Time.realtimeSinceStartup - promiseByContextRequestedTime.Value) > REQUESTS_TIME_OUT)
+                if ((Time.realtimeSinceStartup - promiseByContextRequestedTime.Value) > REQUESTS_TIME_OUT_SECONDS)
                 {
-                    ResolvePendingWearablesByContextPromise(
-                        promiseByContextRequestedTime.Key,
-                        null,
-                        $"The request for the wearable context '{promiseByContextRequestedTime.Key}' has exceed the set timeout!");
-                    expiredRequestedTimes.Add(promiseByContextRequestedTime.Key);
+                    expiredRequests.Add(promiseByContextRequestedTime.Key);
                 }
             }
 
-            foreach (var expiredTimeToRemove in expiredRequestedTimes)
+            foreach (var expiredRquestToRemove in expiredRequests)
```

Comment:
```suggestion
            foreach (var expiredRequestToRemove in expiredRequests)
```

Reply:
Done in https://github.com/decentraland/explorer/pull/2300/commits/a15f01f125788ef879a02ca6446d093b4dced3fb. Thanks!